keps/0007-kic-performance-test.md

```diff
@@ -0,0 +1,557 @@
+---
+title: KIC Performance Test Solution
+status: implementable
+---
+
+# Notes
+For references see the dependencies related to this proprosal to check the progress of related efforts:
+- solution 
+- deployment and configuration
+- code/repo
+- configuration and execution
+- profiling (code profiler)
+- histogram execution time and calculates percentiles
```

Comment:
I'm not sure I understand this notes sections, perhaps we can add URL links?

Reply:
similar as https://perf.dashboard.istio.io/benchmarks/cpu_vs_conn/

Reply:
Ok... still not sure I understand these notes HOWEVER they are just notes and I don't want to block on them, we can always update later.

Reply:
hmm... the execution time of request, API, which basically a metrics diagram, similar as profiling.